git/remote.py

```diff
@@ -391,7 +391,7 @@ def __init__(self, repo, name):
 
     def __getattr__(self, attr):
         """Allows to call this instance like
-        remote.special( \*args, \*\*kwargs) to call git-remote special self.name"""
+        remote.special( \\*args, \\*\\*kwargs) to call git-remote special self.name"""
```

Comment:
BTW, FWIW and all that, I don't know why the asterisks in this particular docstring are backslash escaped in the first place.